add usuful link

Futsum1 opened the pull request:
## Purpose
updated pull request and provided more resource 

## Approach 
I added more documents, DevOps resource links, and Git cheat sheet  to the README.md 

## Learning
hands-on experience with a fundamental aspect of collaboration on Github pull request

Closes #6

Futsum1 committed:
add usuful link

Futsum1 committed:
update

Futsum1 committed:
This is my fourthPR

Futsum1 committed:
Merge branch 'development' into mac-6-add-sp-link

Futsum1 committed:
update

Futsum1 committed:
update

fedluk70 reviewed (commented):
good job F

Futsum1 committed:
conflict resolved

Futsum1 committed:
update

Futsum1 commented:
*------*

fedluk70 commented:
nice job

demewozshiferaw committed:
Merge branch 'development' into mac-6-add-sp-link

anjuyeddu reviewed (commented):
great Github resources

michaelachrisco commented on README.md:
```diff
@@ -2,6 +2,7 @@
 
 
 
+## 
```
What is this line doing? Do we need this line in the markdown?

Futsum1 replied:
Not at all, The branch name shouldn't be at the "Read me" I'm not sure how i added it there, it was probably by mistake.

michaelachrisco replied:
All good, thanks for removing those references! Maybe we had a merge conflict at some point? Either way, looks good now. Thanks for taking care of that.

michaelachrisco commented on README.md:
```diff
@@ -12,6 +13,17 @@
 
 *All combined notes, links and documents are listed below:
 
+mac-6-add-sp-link
```
Lets remove the branch names within the documentation.

michaelachrisco commented on README.md:
```diff
@@ -71,4 +83,8 @@
+#### PULL REQUESTS FOR BRAGGING RIGHTS 
+
 
+ development
```
Same, lets remove the branch name from the documentation.

michaelachrisco commented on README.md:
```diff
@@ -71,4 +83,8 @@
   
 
 
+mac-6-add-sp-link
```
Same here. Lets remove the branch name from the pull request.

michaelachrisco commented on README.md:
```diff
@@ -12,6 +13,17 @@
 
+mac-6-add-sp-link
+#### [Github Documentation](https://docs.github.com/en/github)
+### [Git and Github tutorials on Codecademy](https://www.codecademy.com/learn/learn-git)
```
Why does this link have a different header than the other links?

Futsum1 replied:
Hi Michael, That happened by mistake, I just fixed. All has to be the same. Thanks for that

michaelachrisco commented on README.md:
```diff
@@ -12,6 +13,17 @@
 *All combined notes, links and documents are listed below:
 
+mac-6-add-sp-link
+#### [Github Documentation](https://docs.github.com/en/github)
```
I really like the inclusion of the original docs!

Ruhama21 replied:
_Thank you for your help. michaelchrisco!_

michaelachrisco reviewed (changes requested):
Awesome job on your first PR @Futsum1! I looked at the links to make sure they do not 404 and they all are really good resources. I particularly like the inclusion of the `git` resources. Nice!

Couple of blockers: 
1. Lets remove Headers markdown IE `##` that does not contain a header. 
2. Lets remove branch names from the documentation. The branch names are not useful to know outside the pull request.

Futsum1 committed:
Update README.md

Hi Micheal, sorry for the delay, I removed the blockers that you mentioned, all looks good my side.

Futsum1 committed:
Merge branch 'development' into mac-6-add-sp-link

michaelachrisco commented on README.md:
```diff
@@ -13,6 +15,14 @@
+#### [Youtube tutorials on Git and Github](https://www.youtube.com/results?search_query=git+github+tutorial.)
+
+
+## This is my fourth PR
```
I believe this line is the last thing that needs to be removed and we should be good!

Futsum1 replied:
Thanks, I removed the last line.

michaelachrisco reviewed (changes requested):
Thanks for making the requested changes @Futsum1 ! Looks good. 

One last blocker, looks like there is one last line that needs to be removed, and we should be good.

Futsum1 committed:
Update README.md

fedluk70 reviewed (commented):
Looks good

Ruhama21 reviewed (commented):
Thank you for letting me to review changes! please Add more link!

michaelachrisco reviewed (approved):
Woo!! Everything looks good, thanks for everything @Futsum1 !

Futsum1 committed:
added more usefullink

Futsum1 committed:
Resolved merge conflicts

Futsum1 committed:
Update README.md

Futsum1 committed:
Update README.md

Futsum1 commented:
added more links
